### PR TITLE
[snap] switch to core18 base

### PR DESCRIPTION
**Pull request by Saviq**

### CODE REVIEW

**Comment by Saviq** [comment]
I left
```
The 'qemu' part needs the following libraries that are not included in the snap or base: 
usr/lib/x86_64-linux-gnu/libGL.so.1
usr/lib/x86_64-linux-gnu/libGLU.so.1
usr/lib/x86_64-linux-gnu/libGLX.so.0
usr/lib/x86_64-linux-gnu/libGLdispatch.so.0
These dependencies can be satisfied via more stage-packages, more parts, or content sharing.
```
On purpose, since we don't use GL in our VMs.

**Comment by Saviq** [comment]
Looks like I need to fix CI, snapcraft doesn't support bases and `lxd` yet.

**Comment by Saviq** [comment]
Moved to base on #594 to avoid conflict with the move to `xenial`.

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -163,13 +89,12 @@ parts:
     - on armhf: [qemu-system-arm]
     - on arm64: [qemu-system-arm]
     - qemu-utils
-    - libslang2
```
Comment (gerboland) [inline snap/snapcraft.yaml]: I've had to re-introduce this dependency when I merged this with my strict confinement stuff, qemu needs it:

Feb 06 13:56:09 box2 multipassd[19512]: /snap/multipass/x2/usr/bin/qemu-system-x86_64: error while loading shared libraries: libslang.so.2: cannot open shared object file: No such file or directory

Why did you remove?

Comment (Saviq) [reply snap/snapcraft.yaml]: `snapcraft` didn't complain about it, but I'll bring it back then.

**Comment by Saviq** [comment]
bors try

File: snap/snapcraft.yaml (hunk 1 above)
Comment (townsend2010) [reply snap/snapcraft.yaml]: Just a note, `libslang2` was originally added to fix the snap on Solus Linux.

**Comment by Saviq** [comment]
bors try

**Comment by Saviq** [comment]
bors try

**Comment by Saviq** [comment]
bors try

**Comment by Saviq** [comment]
bors cancel

**Comment by Saviq** [comment]
bors try

**Comment by Saviq** [comment]
bors try

**Comment by Saviq** [comment]
bors try

**Comment by Saviq** [comment]
@townsend2010 other than discarding the snapshot... and in the absence of snap epochs... not sure :/

**Comment by Saviq** [comment]
bors try

**Comment by Saviq** [comment]
bors try

**Comment by Saviq** [comment]
wth
bors try

**Comment by ricab** [comment]
LOL what is going on here? Why do you expect retrials to work?

**Comment by Saviq** [comment]
bors try

**Comment by Saviq** [comment]
bors try

**Comment by Saviq** [comment]
bors try

**Comment by Saviq** [comment]
bors try

**Comment by Saviq** [comment]
bors try

**Comment by Saviq** [comment]
bors try

**Comment by Saviq** [comment]
bors try

**Comment by Saviq** [comment]
bors try

**Comment by Saviq** [comment]
bors try

**Comment by Saviq** [comment]
This PR has lived past its usefulness..
